lib/browser/api/ipc-main.js

```diff
@@ -1,16 +1,3 @@
-const EventEmitter = require('events').EventEmitter
+const {EventEmitter} = require('events')
 
-const emitter = new EventEmitter()
-
-const removeAllListeners = emitter.removeAllListeners.bind(emitter)
-emitter.removeAllListeners = function (...args) {
-  if (args.length === 0) {
-    throw new Error('Removing all listeners from ipcMain will make Electron internals stop working.  Please specify a event name')
-  }
-  removeAllListeners(...args)
-}
-
-// Do not throw exception when channel name is "error".
-emitter.on('error', () => {})
```

Comment:
anybody know why do we need this piece of code?

Reply:
@miniak Node.JS throws an unhandled exception if an event emitter emits `error` with no handlers.  Having this handler stops that occuring.  We definitely still need it 👍

Reply:
@MarshallOfSound restored

Reply:
https://github.com/nodejs/node/blob/master/lib/events.js#L149

Reply:
@MarshallOfSound `ipcRenderer` does not handle the `error` event compared to `ipcMain`